BeeAppy/src/screens/Home.js

```diff
@@ -12,6 +15,7 @@ export default class Home extends React.Component {
   };
   render() {
     const { navigate } = this.props.navigation;
```

Comment:
Can we remove rendering of current lat/long?  Don't think it's relevant to user :)